### DISCUSSION TITLE
Incompatibility with Magento 2.4.6-p6 (latest security patch)

### DISCUSSION THREAD

**Discussion by pabodah** [breezefront/community#286] [5. General]
Hello,

Seems like there is an incompatibility with the latest Magento version 2.4.6-p6

Before the upgrade
[Screencast 2024-06-12 23-01-30.webm](https://github.com/breezefront/community/assets/1543867/5079d485-ee9e-495e-b879-a5d752b1ed56)

In checkout > shipping, “Ship here” button is working and Shipping methods can be checked

After the upgrade

[Screencast 2024-06-12 23-02-13.webm](https://github.com/breezefront/community/assets/1543867/7a2a6cda-d0dc-4e98-9019-d1bbcf694ec4)

In checkout > shipping, “Ship here” button is NOT working and Shipping methods cannot be checked

**Comment by vovayatsyuk** [comment]
I can't reproduce this issue on the latest 2.4.7 version. Please provide URL to your site so I can check what is wrong with css.
